src/handlers/handleOptionChange/handleOptionChange.tsx

```diff
@@ -0,0 +1,20 @@
+interface CurrentOptionHandler {
+  event: React.ChangeEvent<HTMLSelectElement>;
+  setPerPage: (pageNumber: number) => void;
+  setCurrentPage: (pageNumber: number) => void;
+  newParams: URLSearchParams;
+  setSearchParams: (currentOptions: URLSearchParams) => void;
+}
+
+export default function handleOptionChange(params: CurrentOptionHandler): void {
+  const { event, setPerPage, setCurrentPage, newParams, setSearchParams } =
+    params;
+  const startPage = import.meta.env.VITE_START_PAGE;
```

Comment:
The `startPage` is being assigned from `import.meta.env.VITE_START_PAGE`. Ensure that this environment variable is correctly set and accessible in your environment configuration to avoid unexpected behavior.